src/vs/workbench/api/node/extHost.api.impl.ts

```diff
@@ -199,6 +199,9 @@ export function createApiFactory(
 			getLanguages(): TPromise<string[]> {
 				return extHostLanguages.getLanguages();
 			},
+			getLanguage(resource: vscode.Uri, position?: vscode.Position): TPromise<string> {
+				return extHostLanguages.getLanguage(<URI>resource, position ? { lineNumber: position.line, column: position.character } : undefined);
```

Comment:
That won't work because the API-position is zero-based, the internal-position is 1-based. Use `extHostTypeConverter` here.